build.sbt

```diff
@@ -33,35 +33,23 @@ lazy val settings = Seq(
 
                                     /** projects */
 lazy val root = project
-  .in(file("."))
```

Comment:
I think we need to have root to be able run `sbt tasks` on root and all aggregated projects. [Ref](https://www.scala-sbt.org/0.13/docs/Multi-Project.html#Aggregation)

Reply:
It should be smth like [here](https://github.com/NikitaZhevnitskiy/sbt-multimodule-example/blob/master/build.sbt) or [here kafka-security](https://github.com/simplesteph/kafka-security-manager/blob/master/build.sbt#L9)